resume/src/components/Footer.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+const Footer = (props) => {
+    const awards = props.awards.map(item =>
+        <div className="award">
+            <div>{item.title} | {item.awarder} | {item.date}</div>
+            <div>{item.summary}</div>
+        </div>
+    )
+    const publications = props.publications.map(item =>
+        <div className="publication">
+            <div><a href={item.website} alt={item.name}>{item.name}</a> | {item.publisher} | {item.releaseDate}</div>
+            <div>{item.summary}</div>
+        </div>
+    )
+    const languages = props.languages.map(item =>
+        <span>{item.language}, {item.fluency} </span>
+    )
+    const interests = props.interests.map(item =>
+        <span>{item.name}, </span>
+    )
+
+    const references = props.references.map(item =>
+        <div>{item.name}, {item.reference}</div>
+    )
+
+    return (
+        <div className="footer">
```

Comment:
can further DRY the code with a reusable component that takes in the label value for the h3 tag, and content inside the div:

```jsx
const FooterSection = (({ label, content }) => 
  <div>
    <h3>{label}</h3>
    {content}
  </div>
)
```

Reply:
Great idea, thank you!